activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb

```diff
@@ -358,33 +358,33 @@ def insert_fixture(fixture, table_name)
       def insert_fixtures(fixtures, table_name)
```

Comment:
We can't kill/change this because it's documented, but we can deprecate it. Especially as it is itself fairly new, I don't think we have any strong motivation to keep it long term.

Reply:
Done in my latest commit